spec/active_hash/base_spec.rb

```diff
@@ -292,6 +292,27 @@ class Region < ActiveHash::Base
       record.id.should == 1
       record.name.should == 'US'
     end
+
+    it "finds the record with the specified id as a string" do
+      record = Country.find_by(:id => 1)
+      record.name.should == 'US'
+    end
+  end
+
+  describe ".match_options?" do
```

Comment:
since `.match_options?` is a private method, you may want to implement this test from `find_by`.

That way if the internal implementation changes (and the external implementation stays the same), the tests won't have to change.